### PR TITLE
Vote entity addded

### PR DESCRIPTION
**Pull request by Zap1999**

### CODE REVIEW

File: src/main/java/com/softserve/academy/studhub/entity/Vote.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "votes")
+public class Vote {
```
Comment (DaNkOLULzz) [inline src/main/java/com/softserve/academy/studhub/entity/Vote.java]: HashCode and Equals?

Comment (Zap1999) [reply src/main/java/com/softserve/academy/studhub/entity/Vote.java]: ok ok ok

**Comment by OlhaLozinska** [comment]
good

File: src/main/java/com/softserve/academy/studhub/entity/Vote.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "value")
+    @NotNull
```
Comment (volodymyroseredchuk) [inline src/main/java/com/softserve/academy/studhub/entity/Vote.java]: change @NotNull to nullable = false

**Pull request merged**